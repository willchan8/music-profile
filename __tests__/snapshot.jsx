import { render } from "@testing-library/react";
import Home from "../pages/index";

it("renders home page unchanged", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
