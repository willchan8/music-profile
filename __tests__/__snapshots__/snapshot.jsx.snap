// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <main
    class="main"
  >
    <a
      class="className"
      href="/spotify"
    >
      <button
        class="button"
      >
        Go to Spotify App
      </button>
    </a>
  </main>
</div>
`;
